packages/pwa/app/routes.jsx

```diff
@@ -81,30 +78,37 @@ const routes = [
         exact: true
     },
     {
-        path: '/:locale/cart',
+        path: '/cart',
         component: Cart,
         exact: true
     },
     {
-        path: '/:locale/product/:productId',
+        path: '/product/:productId',
         component: ProductDetail
     },
     {
-        path: '/:locale/search',
+        path: '/search',
         component: ProductList
     },
     {
-        path: '/:locale/category/:categoryId',
+        path: '/category/:categoryId',
         component: ProductList
     },
     {
-        path: '/:locale/account/wishlist',
+        path: '/account/wishlist',
         component: Wishlist
     },
+    {
+        path: '/:locale',
+        component: Home,
+        exact: true
+    },
     {
         path: '*',
         component: PageNotFound
     }
 ]
 
-export default routes
+const modifiedRoutes = modifyRoutesWithUrlConfig(routes, [HOME_HREF, '/callback', '/:locale', '*'])
```

Comment:
ANC (Another naming comment): 

Not in love with the word "modified/modify", it doesn't tell me much about what is happening, almost makes it sound like we are doing something hacky or working around issues we created for ourself. 

`const configuredRoutes = configureRoutes(routes, { ignorePaths: [HOME_HREF, '/callback', '/:locale', '*']})`

The above example give the developer at a glance what is happening so they don't have to go poking around in code. (Noticed that I not only changed the variable name and function name, but changed the signature to accept an object so that we can expand on it in the future. Imaging passing in a custom config object)